internal/blockdb/test_case.go

```diff
@@ -29,8 +29,9 @@ func CreateTestCase(ctx context.Context, db *sql.DB, testName, gitSha string) (*
 
 // AddChain tracks and attaches a chain to the test case.
 // The chainID must be globally unique. E.g. osmosis-1001, cosmos-1004
```

Comment:
I know this line was already here, but would this be more accurate as `must be unique within the test case`?